mlflow/server/js/src/experiment-tracking/actions/ModelGatewayActions.ts

```diff
@@ -45,11 +43,19 @@ export const queryModelGatewayRouteApi = (
   route: ModelGatewayRoute,
   data: ModelGatewayQueryPayload,
 ) => {
+  
+  const { inputText } = data;
+  const textPayload = ModelGatewayService.createEvaluationTextPayload(inputText, route);
+  const processed_data = {
+    ...textPayload,
+    ...data.parameters,
+  };
```

Comment:
databricks implementation created the proper payload but the oss one does not https://github.com/databricks/universe/pull/373447